tests/unit/lib/test_opensearch_relation_provider.py

```diff
@@ -303,7 +311,10 @@ def add_dashboard_relation(self):
         self.harness.update_relation_data(
             opensearch_relation,
             f"{DASHBOARDS_CHARM}",
-            {"requested-secrets": '["username", "password", "tls", "tls-ca", "uris"]'},
+            {
+                "index": "opensearch-dashboards-test",
+                "requested-secrets": '["username", "password", "tls", "tls-ca", "uris"]',
```

Comment:
There is a new check on the `data_interfaces` that fails if `index` is not set on the relation.